mne/minimum_norm/tests/test_resolution_matrix.py

```diff
@@ -114,13 +116,71 @@ def test_resolution_matrix():
     assert_array_almost_equal(rm_mne, rm_mne.T)
     assert_array_almost_equal(rm_mne_free, rm_mne_free.T)
 
-    # Test conversion to STC
+    # Some arbitrary vertex numbers
     idx = [1, 100, 400]
-    stc_psf = get_point_spread(rm_mne, forward_fxd['src'], idx, norm=True)
-    stc_ctf = get_cross_talk(rm_mne, forward_fxd['src'], idx, norm=True)
 
-    assert_array_almost_equal(stc_psf.data, stc_ctf.data)
+    # check various summary and normalisation options
+    for mode in [None, 'sum', 'mean', 'maxval', 'maxnorm', 'svd']:
+
+        n_comps = [1, 3]
+
+        if mode in [None, 'sum', 'mean']:
+
+            n_comps = [1]
+
+        for n_comp in n_comps:
+
+            for norm in [None, 'max', 'norm', True]:
+
+                stc_psf = get_point_spread(
+                    rm_mne, forward_fxd['src'], idx, mode=mode, n_comp=n_comp,
+                    norm='norm', return_svd_vars=False)
+                stc_ctf = get_cross_talk(
+                    rm_mne, forward_fxd['src'], idx, mode=mode, n_comp=n_comp,
+                    norm='norm', return_svd_vars=False)
+
+                # for MNE, PSF/CTFs for same vertices should be the same
+                assert_array_almost_equal(stc_psf.data, stc_ctf.data)
+
+    # check SVD variances
+    stc_psf, s_vars_psf = get_point_spread(
+        rm_mne, forward_fxd['src'], idx, mode=mode, n_comp=n_comp,
+        norm='norm', return_svd_vars=True)
+    stc_ctf, s_vars_ctf = get_cross_talk(
+        rm_mne, forward_fxd['src'], idx, mode=mode, n_comp=n_comp,
+        norm='norm', return_svd_vars=True)
+
+    assert_array_almost_equal(s_vars_psf, s_vars_ctf)
+
+    assert s_vars_psf[0] > s_vars_psf[1] > s_vars_psf[2]
```

Comment:
If it's a variance it should sum to 1.
```suggestion
    assert s_vars_psf[0] > s_vars_psf[1] > s_vars_psf[2]
    assert_allclose(s_vars_psf.sum(), 1.)
```

Reply:
Good point. Actually 100 as it's percent. 
Is there a preference for fraction/percent/variance? I think percent is the most intuitive.